recipes/is2view/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "IS2view" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "a6d9d02c6bd993ad128e8f5c1dc502c956e36cb81a63e121316cc43a4f23f632" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+    - setuptools
+  run:
+    - python >=3.6
+    - ipyleaflet
+    - matplotlib-base
+    - netcdf4
+    - numpy
+    - rioxarray
+    - setuptools-scm
+    - setuptools
```

Comment:
These are probably a mistake upstream, I don't think `setuptools-scm` and `setuptools` are needed at run time. 

```suggestion
```